feature-libs/product/configurators/rulebased/cpq/cpq-configurator-rest.adapter.ts

```diff
@@ -1,25 +1,18 @@
-import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
-import {
-  CartModification,
-  ConverterService,
-  OccEndpointsService,
-} from '@spartacus/core';
+import { CartModification } from '@spartacus/core';
 import { CommonConfigurator } from '@spartacus/product/configurators/common';
-import { Observable } from 'rxjs';
-import { map } from 'rxjs/operators';
+import { Observable, of } from 'rxjs';
+import { delay, map } from 'rxjs/operators';
 import { RulebasedConfiguratorAdapter } from '../core/connectors/rulebased-configurator.adapter';
 import { Configurator } from '../core/model/configurator.model';
-import { CpqAccessLoaderService } from '../occ/cpq/cpq-access-loader.service';
+import { CpqConfiguratorRestService } from './cpq-configurator-rest.service';
 
 @Injectable()
 export class CpqConfiguratorRestAdapter
```

Comment:
Test coverage is way too low (64 % statement coverag)

Reply:
deleted some unused dummy coding an added an additional test. should be 100% now